docs/src/content/docs/guides/authoring-content.md

```diff
@@ -202,9 +202,140 @@ var fun = function lang(l) {
 ```
 ````
 
-```md
-Long, single-line code blocks should not wrap. They should horizontally scroll if they are too long. This line should be long enough to demonstrate this.
-```
+### Expressive Code features
+
+Starlight uses [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) to extend formatting possibilities for code blocks.
+Expressive Code’s text markers and window frames plugins are enabled by default.
+Code block rendering can be configured using Starlight’s [`expressiveCode` configuration option](/reference/configuration/#expressivecode).
+
+#### Text markers
```

Comment:
How standard is the phrase "text markers" for what seems to be describing "highlighting"? I don't think I use this phrase myself at all, so I might be tempted to rename?

```suggestion
#### Line and Text highlighting
```

Reply:
I also considered this — and “highlight” is used as a verb in this section. I think the slight difficulty is that “syntax highlighting” has such a specific meaning that I wasn’t sure if “highlighting” in this context might imply _that_, rather than what we might think of as “highlighting” in a word processing or handwritten context. Do you think that could be an issue?

Reply:
I went through the same struggle when naming this feature and the plugin providing it. The unwanted proximity to the widely known and used term "syntax highlighting" was the deciding factor for me that I had to find another name, so I ended up with "text markers".